### DISCUSSION TITLE
Feature Request: USB Keyboard Layout Change Option, Including ABNT 2

### DISCUSSION THREAD

**Discussion by helisondjo** [rohitsangwan01/uni_control_hub#34] [Ideas]
Dear Uni Control Hub Development Team,

I hope this message finds you well. I am writing to suggest a feature that I believe would greatly enhance the usability of the Uni Control Hub application. Specifically, I would like to request the addition of an option to change the keyboard layout via USB, with a particular emphasis on including the ABNT 2 layout.

The ABNT 2 layout is widely used in Brazil and includes specific keys that are essential for typing in Portuguese.

Here are the key benefits of this feature:

Enhanced Accessibility: Users can easily switch to their preferred keyboard layout, making the application more accessible to non-English speakers.
Increased Productivity: Typing in the native layout can reduce errors and increase typing speed, leading to better productivity.
User Satisfaction: Providing support for widely used layouts like ABNT 2 can increase user satisfaction and loyalty.
I believe that this feature would be a valuable addition to Uni Control Hub and would greatly appreciate your consideration of this suggestion.

Thank you for your time and attention.

Best regards,
Heli

**Comment by rohitsangwan01** [comment]
@helisondjo Thak you for the suggestion, will try to implement multiple keyboard layout support in my free time
